### ISSUE TITLE
Can’t Edit/Select Older Generations

### ISSUE THREAD

**Issue by Bakugonaomygod** [FlagBrew/PKSM#1435]
Can’t  inject Pokémon into several generations included HG, SS etc and doesn’t respond to hitting any buttons.
If it’s not obvious I know nothing about this stuff so please stupid vise me.

![IMG_9045](https://github.com/FlagBrew/PKSM/assets/156156001/af59f500-7001-426e-9967-2b195b91d398)

**Comment by SpiredMoth** [comment]
You need to [point PKSM to saves](https://github.com/FlagBrew/PKSM/wiki/Settings#extra-saves) for anything to show up on those "Absent Games" screens.

**Comment by Bakugonaomygod** [comment]
Does nothing when I try to do that process. Won’t select options
![IMG_9046](https://github.com/FlagBrew/PKSM/assets/156156001/8d5662e0-7ff9-4445-a7a4-2bba985c6386)

**Comment by SpiredMoth** [comment]
A vague statement of "does nothing" and a screenshot of the instructions I directed you to provides no information on why the process is not working for you.  If you want any more help either provide more information or join [our Discord server](https://discord.gg/bGKEyfY).

**Comment by Bakugonaomygod** [comment]
I have reviewed those Extra Save instructions, followed them to a T which still gives me no response on the step of selecting older games. The instructions specifically had Platimun as the example so of course I tried that and I didn’t have any results. When I say “doing nothing” I’m saying there’s no response to selecting games. I don’t know how to be any more clear about something I don’t understand?
